src/services/Router.ts

```diff
@@ -0,0 +1,48 @@
+export default class Router {
+    path: Location
+    appContainer: HTMLDivElement
+
+    constructor() {
+        this.appContainer = document.querySelector("#app")
+        this.path = window.location
+        this.render()
+    }
+
+
+    initPage = (): void => {
+        switch (this.path.hash) {
+            //main page
+            case("/" || "") : {
```

Comment:
```suggestion
            case "/" :
            case "" : {
```

O ile eslint tego nie zablokuje. Jak zablokuje to wyłącz mu rule za to odpowiedzialny